reference/release-notes/1.23.1.md

```diff
@@ -0,0 +1,76 @@
+---
+orphan: true
+---
+# 1.23.1
+
+These release notes cover new features and changes in Anbox Cloud 1.23.1.
+
+Anbox Cloud 1.23.1 is a patch release. To understand minor and patch releases, see {ref}`ref-release-notes`.
+
+Please see {ref}`ref-component-versions` for a list of updated components.
+
+## Requirements
+
+See {ref}`ref-requirements` for detailed requirements to run Anbox Cloud 1.23.1.
+
+## New features & improvements
+
+### Anbox Cloud Appliance (epoch=1)
+
+* You can now change the appliance network configuration using `anbox-cloud-appliance config set`. You can change the public IP address and location of the appliance.<!--AC-2727-->.
+* You can now configure the [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for those API endpoints that the appliance reverse proxy passes to the Anbox Stream Gateway. You can set the following using the `anbox-cloud-appliance config set` command:<!--AC-2732-->
+  - The network's public IP address (`network.public_address`)
+  - The network's DNS name (`network.location`)
+  - The [HTTP origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) (`core.https_allowed_origin`)
+  - List of allowed HTTP headers (`core.https_allowed_headers`)
+  - List of allowed HTTP methods (`core.https_allowed_methods`)
+* You can skip setting up coturn when initialising the appliance and can configure a custom STUN server.<!--AC-2719-->
+* The [`prepare-node-script`](prepare-node-script.md) command is extended to support systems with an already installed NVIDIA driver. <!--AC-2714-->
+
+### Streaming
+
+* With the 1.23.0 release, the functionalities of the *Instances* page and the *Sessions* page on the web dashboard were merged. For debugging purposes, viewing a list of available sessions can be really useful. With the 1.23.1 release, you can list all available sessions using the `anbox-stream-gateway sessions list` command.<!--AC-2662-->
+* You can dynamically change the display density when joining an existing streaming session.<!--AC-2673-->
+
+### Dashboard
+
+* The micromatch dependency in the Anbox Cloud dashboard is updated to include an upstream fix for [CVE-2024-4067](https://github.com/advisories/GHSA-952p-6rrq-rcjv).<!--AC-2709-->
+
+### VHAL
+
+* Starting with 1.23.0,the VNDK version required for building a custom VHAL is 34.<!--AC-2720-->
+
+### Android vendor image
+
+* WiFi support for Android 12 vendor images.<!--AC-2707-->
+
+### Other
+
+* Android security updates for September 2024 (see [Android Security Bulletin - September 2024](https://source.android.com/docs/security/bulletin/2024-09-01) for more information). <!--AC-2712-->
+* The Android WebView has been updated to [128.0.6613.127](https://chromereleases.googleblog.com/2024/09/chrome-for-android-update.html).
+
+## Known issues
+
+* WiFi support is not yet available with the Android 14 vendor image. <!--AC-2647-->
```

Comment:
This's not true. The wifi support has been supported since this patch(1.23.2) release.

Reply:
I think you mean 1.23.1 release. You're right in that case. This was a known issue for 1.23.0, I will remove it, thanks!